boards/intel/btl/Kconfig.defconfig

```
@@ -0,0 +1,56 @@
+# Copyright (c) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+config BUILD_OUTPUT_STRIPPED
+	default y
+
+config MP_MAX_NUM_CPUS
+	default 2
+
+# TSC on this board is 2.9 GHz, HPET is 19.2 MHz
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 2900000000 if APIC_TSC_DEADLINE_TIMER
+	default 2900000000 if APIC_TIMER_TSC
+	default 19200000
+
+if APIC_TIMER
+config APIC_TIMER_IRQ
+	default 24
+endif
```

```suggestion
if APIC_TIMER

config APIC_TIMER_IRQ
	default 24

endif
```
fix styling throughout